_projects/use_coala.md

```diff
@@ -74,13 +74,13 @@ can be used to help select a repository that meets those criteria.
 To filter out repositories that do not meet the first two requirements, combine
 
 1. [`stars:>99`](https://github.com/search?o=desc&q=stars%3A%3E%3D99&ref=searchresults&s=updated&type=Repositories&utf8=%E2%9C%93), and
-2. [`pushed:>2016-09-30`](https://github.com/search?utf8=%E2%9C%93&q=pushed%3A%3E2016-09-30&type=Repositories&ref=searchresults)
+2. [`pushed:>2017-09-30`](https://github.com/search?utf8=%E2%9C%93&q=pushed%3A%3E2017-09-30&type=Repositories&ref=searchresults)
 
-The [combined query](https://github.com/search?o=asc&q=stars%3A%3E%3D100+pushed%3A>2016-09-30&ref=searchresults&s=updated&type=Repositories&utf8=%E2%9C%93)
+The [combined query](https://github.com/search?o=asc&q=stars%3A%3E%3D100+pushed%3A>2017-09-30&ref=searchresults&s=updated&type=Repositories&utf8=%E2%9C%93)
 has more than 28,000 repository results, so there are many to choose from.
 
 An optional additional search term is
-[`forks:>100`](https://github.com/search?o=asc&q=stars%3A%3E%3D100+pushed%3A%3E2016-09-30+forks%3A%3E%3D100&ref=searchresults&s=updated&type=Repositories&utf8=%E2%9C%93)
+[`forks:>100`](https://github.com/search?o=asc&q=stars%3A%3E%3D100+pushed%3A%3E2017-09-30+forks%3A%3E%3D100&ref=searchresults&s=updated&type=Repositories&utf8=%E2%9C%93)
```

Comment:
what is this link referring to actually? why have you given the date of 30th September 2017 thats so far ahead in the future and GitHub is still showing results. Shouldn't you be giving present date or something?

Reply:
The link is working fine, it refers to combined query = star:>99,  pushed :>2017-09-30 and forks:>100 as given in https://github.com/coala/projects/blob/master/_projects/use_coala.md

Reply:
what does  pushed :>2017-09-30  mean?

Reply:
it means the repository has atleast one push after 30th September, 2017

Reply:
shit got confused between 2017 and 18 🤪.  You are all good